### DISCUSSION TITLE
Is there a key for showing auto complete?

### DISCUSSION THREAD

**Discussion by Android789515** [helix-editor/helix#9646] [Q&A] [answered]
I noticed auto complete is not showing under some circumstances.  Is there a key I can press to open the menu?

**Comment by the-mikedavis** [comment, marked as answer]
You can manually trigger completion in insert mode with `C-x` (<kbd>Ctrl</kbd>+<kbd>x</kbd>), see https://docs.helix-editor.com/master/keymap.html#insert-mode

**Reply by Android789515** [reply]
Thanks.
